alpha/apps/kaltura/lib/myPlaylistUtils.class.php

```diff
@@ -1427,15 +1427,15 @@ public static function getPlaylistDataFromEntries($entries, $flavorParamsIds, $c
 		return array($entryIds, $durations, $mediaEntry, $captionFiles);
 	}
 
-	protected static function getCaptionFilesForEntryIds($entryIds, $captionLanguages)
+	public static function getCaptionFilesForEntryIds($entryIds, $captionLanguages, $allLanguages = false)
```

Comment:
No need to add another flag to the function, we can set that $captionLanguages is default to null, and if it is not set that we grab all langueges

Reply:
I'm removing this change, it isn't needed anymore (was needed before and forgot to remove it)